src/meds_etl/omop.py

```diff
@@ -188,6 +194,8 @@ def write_event_data(
                 for option in options
                 if table_name + option in schema.names()
             ]
+            # We prefer user defined time field options over the default time options if available
```

Comment:
Why not:
```
            options = table_details.get("time_field_options", []) + ["_start_datetime", "_start_date", "_datetime", "_date"]
            options = [
                cast_to_datetime(schema, table_name + option, move_to_end_of_day=True)
                for option in options
                if table_name + option in schema.names()
            ]
```

Reply:
@EthanSteinberg I like your suggestion cos it's more concise. Actually, the fields defined `time_field_options` only contain the table name e.g. `visit_end_datetime` and `visit_end_date`, I can change it to `_end_datetime` and `_end_date` so your suggestion would work, what do you think?

Reply:
Hmm. Yeah that would work. I would also be down with changing the code above to:

ptions = table_details.get("time_field_options", []) + [table_name + "_start_datetime", table_name + "_start_date", table_name + "_datetime", table_name + "_date"]